internal/btf/core.go

```diff
@@ -31,7 +33,14 @@ func (f COREFixup) String() string {
 	if f.Poison {
 		return fmt.Sprintf("%s=poison", f.Kind)
 	}
-	return fmt.Sprintf("%s=%d->%d", f.Kind, f.Local, f.Target)
+	if f.Validate {
+		return fmt.Sprintf("%s=%d->%d", f.Kind, f.Local, f.Target)
+	}
+	return fmt.Sprintf("%s=%d", f.Kind, f.Target)
+}
+
+func (f COREFixup) validate() bool {
```

Comment:
I do not see this function used anywhere, is it only on my side?

Reply:
oops, leftover from earlier. I'll remove it.